src/utils/breadcrumb-utils.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Badge, BreadcrumbItem } from '@patternfly/react-core';
+import styles from '@patternfly/react-styles/css/components/Breadcrumb/breadcrumb';
+import { ApplicationSwitcher } from '../components/ApplicationDetails/ApplicationSwitcher';
+import { WorkspaceSwitcher } from '../components/ApplicationDetails/WorkspaceSwitcher';
+import { useWorkspaceInfo } from './workspace-context-utils';
+
+export const useApplicationBreadcrumbs = (appDisplayName = null, withLink = true) => {
+  const params = useParams();
+  const applicationName = params.appName;
+
+  const { workspace } = useWorkspaceInfo();
+
+  return [
+    <Badge key="badge" isRead>
+      WS
+    </Badge>,
+    <span key="badge-divider" className={styles.breadcrumbItemDivider} />,
+    <BreadcrumbItem key="workspace-link" to="#">
+      <Link className="pf-c-breadcrumb__link" to="#">
+        {workspace}
```

Comment:
This is a link but doesn't take me anywhere. Maybe this one can also take user to application list page until we have a workspace list page.


https://user-images.githubusercontent.com/6041994/219340970-7ebd6339-3a52-4cb2-bc95-926e8d480fd3.mov

Reply:
Sure, For now taking the users to application list page.